dist/bundle.js

```diff
@@ -0,0 +1,743 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+/**
+* TARGET CONSTRUCTOR
+*
+  var settings = {
+	x: 2,
+	y: 2,
+	ttl: 30,
+	color:  ,
+  };
+*
+*/
+
+function Target(settings) {
+	this.x = settings.x || 300;
+	this.y = settings.y || 300;
+	this.r = settings.r || 15;
+	this.ttl = settings.ttl;
+	// No need for color if we use image pattern as fill
+	// this.color = settings.color || '#FF0000';
+	// test
+	this.type = settings.type || "output";
+	this.isScore = settings.isScore || false;
+}
+
+Target.prototype.updateColor = function(dist, n) {
+		this.color = dist;
+		// select specific target
+		// change color attribute
+}
+
+Target.prototype.init = function() {
+	if(this.isScore) {
+	}
+	else {
+		this.drawTarget();
+	}
+}
+
+Target.prototype.drawTarget = function() {
+	var score = 0;
+	var circle = d3.select('#'+this.type+'-svg').append("circle")
+		.attr({
+			"cx": this.x,
+			"cy": this.y,
+			"r": this.r
+		})
+		.style({"fill": "url(#tar_img)"});
+
+	if(this.isScore) {
+		circle.append("text")
+			.text("Score ")
+	}
+}
+
+module.exports = Target;
```

Comment:
We can now use `var Target = require('/relative/path/to/target');` to include all of our objects.
Look at playground.js to see how this works.